### PR TITLE
[MRG] Add code examples for running different searchers on CIFAR10 in the e…

### LINKED ISSUE #450: Add content to nas.md
### Feature Description

Add content to nas.md.
Add code examples for running different searchers on CIFAR10 in the examples folder.
The example should call the CnnModule.
The CIFAR10 dataset should be directly loaded from pytorch. (refer to example code of pytorch on CIFAR10 online)

### PR DESCRIPTION
**Pull request by chengchengeasy**
…xamples folder

This pull request resolves #450 .

The pull request is ready to be merged.

The details of the pull request:
see #450 for more details

### CODE REVIEW

File: examples/nas/cifar10_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+from torch.nn.functional import cross_entropy
+
+from autokeras import CnnModule
+from autokeras.nn.metric import Accuracy
+from nas.greedy import GreedySearcher
+
+if __name__ == '__main__':
+    transform = transforms.Compose(
+        [transforms.ToTensor(),
+         transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
```
Comment (jhfjhfj1) [inline examples/nas/cifar10_tutorial.py]: Please see some examples online. I think they normalized with a different value.
They also did some data augmentation.
Refer to this example.
https://github.com/kuangliu/pytorch-cifar/blob/master/main.py

File: nas/README.md [hunk 2]
```diff
@@ -26,12 +26,15 @@ which may conflict with each other.
...
+4. bayesian optimization: it's the default search strategy of Auto-Keras currently. refer [arXiv:1806.10282](https://arxiv.org/abs/1806.10282) for more detail. 
 
 ## How to run the baseline methods?
 
-Code example containing two parts: data preparation, search.
-Should call CnnModule.
+Refer to examples/nas/cifar10_tutorial.py for more details.
 
 
 ## How to implement your own search?
```
Comment (jhfjhfj1) [inline nas/README.md]: We need content of this part, too.
Let them know which function to override, and what are the meanings of the parameters.
